source/docs/assist/index.markdown

```diff
@@ -16,6 +16,8 @@ Assist is available to use on most platforms that can interface with Home Assist
 
 Did Assist not understand your sentence? [Contribute them.](https://developers.home-assistant.io/docs/voice/intent-recognition/)
 
+If you have opted to not use `default_config:` you must enable the [conversation](https://www.home-assistant.io/integrations/conversation/) integration before you can use Assist.
```

Comment:
Thank you for adding this, @langestefan :muscle: 

I would include the symptom in the description and place the remedy before the cause of the issue.
Also, I think it would make sense to move this section above _Did Assist not understand..._, because it is directly related to the _Look for the Assist icon_ section. 

``suggestion
The Assist icon doesn't show up? Make sure the [conversation](https://www.home-assistant.io/integrations/conversation/) integration is enabled. If you're not using `default_config:`, the conversation integration may not be enabled yet.
```